src/sections/Hero/index.js

```diff
@@ -37,23 +38,27 @@ const Hero = ({ title, actions, social }) => (
       <div className="row">
         <div className="col">
           <div className="action-circle-wrap">
-            {actions.map(({ title, image }, index) => (
-              <div
-                key={`action-${index}`}
-                className={`action-circle bg-${getBg(index)}`}
+            {actions.map(({ join_section_id, title, image }, index) => (
+              <Link
+                duration={500}
+                key={index}
```

Comment:
Don't use just the `index` otherwise we will potentially render multiple components with key="0" or key="N" https://reactjs.org/docs/lists-and-keys.html#keys